app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError, OutdatedVaccineError)

```

Comment:
Place imports on a new line and in an alphabetical order in the future, as follows
```suggestion
from app.errors import (
    OutdatedVaccineError,
    NotVaccinatedError,
    NotWearingMaskError,
)

```

Reply:
thanks